WBooks/Buttons/Buttons.swift

```diff
@@ -14,8 +14,17 @@ extension UIButton {
         gradient = UIButton.wBooksButtonGradient
     }
     
+    func setWBookDisableGradient() {
+        gradient = UIButton.wBooksDisabledButtonGradient
```

Comment:
unnecessary `UIButton`

Reply:
This one does not compile unless I use UIButton!